src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -432,7 +474,10 @@ void CoilCoolingDX::simulate(bool useAlternateMode, Real64 PLR, int speedNum, Re
     // call the simulation, which returns useful data
     // TODO: check the avail schedule and reset data/pass through data as needed
     // TODO: check the minOATcompressor and reset data/pass through data as needed
-    this->performance.simulate(evapInletNode, evapOutletNode, useAlternateMode, PLR, speedNum, speedRatio, fanOpMode, condInletNode, condOutletNode);
+    this->performance.OperatingMode = 0;
+    this->performance.ModeRatio = 0.0;
+    this->performance.simulate(
```

Comment:
I would have preferred to just pass in these values to the performance simulate function, instead of assigning them prior to calling.  This is something that should be addressed as we move forward with the coil work.  But for now, it's working, I'll move on.